src/component/selection/getSelectionOffsetKeyForNode.js

```diff
@@ -17,7 +17,15 @@
  * Get offset key from a node.
```

Comment:
Probably a good idea to flesh out the docblock for this function, now that it does more than just check the current node.